### DISCUSSION TITLE
Controller support

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1478] [Ideas]
add the ability to add a controller key to the .ini, namely the `ConsoleKeyBinding` and `ReloadKeyBinding` entries (based on [ControllerReload by Lemon](https://github.com/justalemon/ControllerReload))

**Comment by kagikn** [comment]
I doubt how much the ability help to be honest, maybe thoughts of other users or script devs would convince me, but I'm not doing without them. I definitely won't add until I make sure #1454 is solved though, because it would need to invoke natives to implement the ability and more users would likely to encounter weird crashes in earlier stages of the game loading without that solving that issue!

**Reply by Michael21107** [reply]
yeah, it's just a little feature, definitely not something to do before closing #1454

**Comment by justalemon** [comment]
As the developer of ControllerReload, this should not be the case due to a limited number of controller buttons (your average keyboard has 104, your average gamepad only 17). ControllerReload is better to be kept standalone due to the way it handles the keybinds.

For the console, you open the console and then you have to go back to the keyboard if you want to type anything on it.
